packages/hint-leading-dot-classlist/package.json

```
@@ -0,0 +1,78 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js",
+      "!dist/tests/**/fixtures/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "@hint/utils-debug": "^1.0.2",
+    "@hint/utils-fs": "^1.0.5",
+    "@hint/utils-i18n": "^1.0.5",
+    "@hint/utils-types": "^1.1.0"
+  },
+  "description": "Inform users about classList.add or classList.remove with leading '.' in the argument pitfall",
+  "devDependencies": {
+    "@hint/parser-javascript": "^3.1.7",
+    "@hint/utils-create-server": "^3.4.6",
+    "@hint/utils-tests-helpers": "^6.3.0",
+    "@types/node": "^14.11.2",
+    "@typescript-eslint/eslint-plugin": "^4.0.1",
+    "@typescript-eslint/parser": "^4.0.1",
+    "ava": "^3.12.1",
+    "cpx": "^1.5.0",
+    "eslint": "^7.8.1",
+    "eslint-plugin-markdown": "^1.0.2",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^15.1.0",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.0.2"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "hint",
+    "leading-dot-classlist",
+    "leading-dot-classlist-hint"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/hint.js",
+  "name": "@hint/hint-leading-dot-classlist",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "@hint/parser-javascript": "^3.0.0",
+    "hint": "^6.0.0"
+  },
+  "repository": {
+    "directory": "packages/hint-leading-dot-classlist",
+    "type": "git",
+    "url": "https://github.com/webhintio/hint.git"
+  },
+  "scripts": {
+    "build": "npm run i18n && npm-run-all build:*",
+    "build-release": "npm run clean && npm run i18n && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
+    "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist",
+    "build:ts": "tsc -b",
+    "clean": "rimraf dist",
+    "i18n": "node ../../scripts/create-i18n.js",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint . --cache --ext .js,.md,.ts --ignore-path ../../.eslintignore",
+    "lint:dependencies": "node ../../scripts/lint-dependencies.js",
+    "lint:md": "node ../../scripts/lint-markdown.js",
+    "test": "npm run i18n && npm run lint && npm run build && npm run test-only",
+    "test-only": "nyc ava",
+    "test-release": "npm run i18n && npm run lint && npm run build-release && ava",
+    "init": "npm install && npm run build",
+    "watch": "npm run build && npm-run-all --parallel -c watch:*",
+    "watch:assets": "npm run build:assets -- -w --no-initial",
+    "watch:test": "ava --watch",
+    "watch:ts": "npm run build:ts -- --watch"
+  },
+  "version": "0.0.1"
+}
```

Add a new line at the end.
```suggestion
}

```